assets/components/headers/roundelHeader/header.jsx

```diff
@@ -11,11 +11,13 @@ import SvgCheckmark from 'components/svgs/checkmark';
 import SvgChevron from 'components/svgs/chevron';
 import SvgGlobe from 'components/svgs/globe';
 import SvgRoundel from 'components/svgs/roundel';
+import SvgGuardianLogo from 'components/svgs/guardianLogo';
 
 // ----- Types ----- //
 
 type PropTypes = {|
   selectedCountryGroup: ?CountryGroup,
+  longLogo?: boolean
```

Comment:
is there a chance we can use the long logo across both forms and delete the old one? AFAIK design was keen to go with the long logo